src/components/avatar/__tests__/Avatar.test.tsx

```diff
@@ -1,12 +1,8 @@
 import React from 'react';
+import { shallow, mount } from 'enzyme';
 import { act } from 'react-dom/test-utils';
 import Avatar from '../Avatar';
-
-function MockAvatarImage() {
-    return <div className="avatar-image" />;
-}
-
-jest.mock('../AvatarImage', () => MockAvatarImage);
```

Comment:
I think this was here so it didn't make an actual network call for the invalid images